ui/app/components/section-tabs.js

```diff
@@ -2,13 +2,12 @@ import Component from '@ember/component';
 
 const SectionTabs = Component.extend({
   tagName: '',
-
   model: null,
   tabType: 'authSettings',
 });
 
 SectionTabs.reopenClass({
-  positionalParams: ['model', 'tabType'],
+  positionalParams: ['model', 'tabType', 'paths'],
```

Comment:
Just a note here that positionParams are going away in Glimmer components, so we'll eventually want to move away from this.